services/net/content/ContentManager.cs

```diff
@@ -114,26 +108,15 @@ public override async Task RunAsync()
                     var kafkaTopics = this.KafkaAdmin.ListTopics();
                     topics = topics.Except(topics.Except(kafkaTopics)).ToArray();
 
-                    // If the topics change we need to subscribe to the new topics.
-                    if (!ContainsAll(this.Topics, topics))
```

Comment:
No need for this as it's performed by the `this.Consumer.Subscribe(topics)` function.